uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/models/leave-balance-report.model.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-env amd */
+
+define([
+  'leave-absences/shared/modules/models',
+  'leave-absences/shared/apis/leave-balance-report.api',
+  'common/models/model'
+], function (models) {
+  'use strict';
+
+  models.factory('LeaveBalanceReport', [
+    '$log', 'Model', 'LeaveBalanceReportAPIMock',
```

Comment:
then make the "real" api service to use the mocked data, this way the model doesn't need to know anything about what's going on, he just assumes that the api service it's getting the data from somewhere